taqueria-plugin-taquito/README.md

```diff
@@ -18,60 +23,60 @@ The target networks, sandboxes, and environments are configured in the Taqueria
 Basic usage is:
 
 ```shell
-taq originate <filename>
+taq originate <contractName>
 ```
 
 ### Basic description
 
-The Taquito plugin exposes a `taq originate` (or `taq deploy`) task in Taqueria which will originate the specified Michelson contract to a Taqueria environment.
+The Taquito plugin exposes a `taq originate` (or `taq deploy`) task in Taqueria which will originate the specified Michelson contract to a Taqueria environment
 
-This will originate a Michelson `.tz` file in the `/artifacts` directory to the default environment (the sandbox named `local`).
+This will originate a Michelson `.tz` file in the `artifacts` directory to the default development environment (the sandbox named `local`)
 
-By default, the storage value used for origination will live in a file named `CONTRACT.default_storage.tz`, in `/artifacts`, where `CONTRACT` is the name of the contract to originate. If it can't find it, it'll not originate the contract. Alternatively, you can explicitly specify a storage file with the `--storage` option described below.
+By default, the storage value used for origination will live in a file named `CONTRACT.default_storage.tz`, in `artifacts`, where `CONTRACT` is the name of the contract to originate. If it can't find it, it will not originate the contract. Alternatively, you can explicitly specify a storage file with the `--storage` option described below
```

Comment:
> If it can't find it, it will not originate the contract.

I wonder if _it_ causes problems here - the end-user might not know what _it_ is referring to here. Thus I'd suggest the following:

If this file does not exist then the contract will not be originated.